src/resolve/metadataResolver.ts

```diff
@@ -297,9 +305,24 @@ export class MetadataResolver {
     return folderName;
   }
 
+  /**
+   * If this file should be considered as a metadata file then return the metadata type
+   */
+  private parseAsMetadata(fsPath: string): string {
+    if (this.tree.isDirectory(fsPath)) {
+      return;
+    }
+    return ['DigitalExperience']
+      .map((type) => this.registry.getTypeByName(type))
+      .find((type) => fsPath.split(sep).includes(type.directoryName))?.name;
+  }
```

Comment:
`parseAsMetadata` isn't really an accurate name.  This is simply checking the file path to see if it contains the `DigitalExperience` directory.  The more special handling code added to this library the less reliable it becomes so we try to avoid it.

Reply:
As an extension to https://github.com/forcedotcom/source-deploy-retrieve/pull/695#discussion_r962779387 I named it `parseAsMetadata` to have outliers where we don't usually consider these as metadata. In the case of DE, it's a json file where we consider the file as a content on other metadata types. I'm open for suggestions though